sdk/python/feast/pyspark/launchers/k8s/k8s.py

```diff
@@ -180,10 +184,6 @@ def _job_from_job_info(self, job_info: JobInfo) -> SparkJob:
             # We should never get here
             raise ValueError(f"Unknown job type {job_info.job_type}")
 
-    def _get_staging_client(self):
-        uri = urlparse(self._staging_location)
-        return get_staging_client(uri.scheme, self._config)
-
     def _get_azure_credentials(self):
```

Comment:
I had changed it recently to just pass in the whole config but if we don't want to do that, then this function needs the Azure blob account name and access key passed in